### ISSUE TITLE
Is there anyway to set content options?

### ISSUE THREAD

**Issue by BillBrower** [techpivot/aws-code-deploy#7]
I'd like to have my deployments overwrite existing content. You can specify this under the advanced options in the AWS CodeDeploy web interface (see below) but I don't see any mention of it in the README, or issues, for this project. 

<img width="819" alt="screen shot 2019-01-23 at 7 03 04 pm" src="https://user-images.githubusercontent.com/1717845/51645290-98e52480-1f41-11e9-884f-8b4e998528f7.png">

**Comment by virgofx** [comment]
Yes. Go ahead and submit a PR and I'll update. Thanks @BillBrower
